requirements.txt

```diff
@@ -1,7 +1,5 @@
 pyodbc
 six
-sqlalchemy
```

Comment:
Probably, but since we do import it directly in `odm2api` I guess it is better to leave it untouched.
We never know if the other dependencies that brings it will drop it in the future.